Simplified/NYPLMyBooksViewController.h

```diff
@@ -5,6 +5,8 @@
 - (id)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil NS_UNAVAILABLE;
 
+- (void)didSelectSync;
```

Comment:
1. I've updated to 'syncMyBooks'
2. For the most part I believe there's a bigger issue with how other classes are requesting a BookRegistry load or sync. I noticed that book lists were not loading after a library switch or fresh user sign in, and since the pull-to-refresh is not always discoverable for every user I was looking for a quick way to invoke that.